src/loaders/ncbi_downloader/ncbi_downloader.py

```diff
@@ -1,31 +1,25 @@
 """
 PROTOTYPE - Download genome files from NCBI FTP server.
 
-usage: ncbi_downloader.py [-h] --download_file_ext
-                          DOWNLOAD_FILE_EXT
-                          [DOWNLOAD_FILE_EXT ...]
-                          --release_ver
-                          [--root_dir ROOT_DIR]
-                          [--source SOURCE]
-                          [--threads THREADS] [--overwrite]
+usage: ncbi_downloader.py [-h] --download_file_ext DOWNLOAD_FILE_EXT [DOWNLOAD_FILE_EXT ...] --release_ver
```

Comment:
I want to make this script requires as few arguments as possible.

Reply:
For the `compute_genome_attribs` script? I used `--source_data_dir` instead there. Otherwise, we have to require both `collection` and `release_ver` arguments.

Reply:
No, for the ncbi downloader. `--release-ver` in the documentation is listed as a boolean flag rather than a parameter with an argument

Reply:
:+1: